meson.build

```diff
@@ -57,6 +58,10 @@ if shellcheck.found()
   test('shellcheck toolbox (deprecated)', shellcheck, args: [toolbox_sh])
 endif
 
+if codespell.found()
+  test('codespell toolbox', codespell, args: ['--skip', meson.project_build_root(), meson.source_root()])
```

Comment:
It should be `meson.project_source_root()`, not `meson.source_root().  See the test failures.